### PR TITLE
fix: issue with the latest version not rendering in IE

### PR DESCRIPTION
**Pull request by mindblender**
Fixes issue with it not rendering in Edge

### CODE REVIEW

File: notification-portlet-webcomponents/notification-icon/src/components/NotificationIcon.vue [hunk 1]
```diff
@@ -32,6 +35,28 @@ import oidc from "@uportal/open-id-connect";
...
+  const edge = ua.includes("Edge/");
+
+  return msie || trident || edge;
+}
+
+const isIE = detectIE();
+
+/**
+ * HACK: This exists because IE/Edge get caught in an infinite event loop when
+ * they try to render bootstrap vue modal or button, this provides a feature
```
Comment (ChristianMurphy) [inline notification-portlet-webcomponents/notification-icon/src/components/NotificationIcon.vue]:
```suggestion
 * they try to render bootstrap vue dropdown, dropdown-item, and dropdown-header, this provides a feature
```

File: notification-portlet-webcomponents/notification-icon/src/components/NotificationIcon.vue [hunk 2]
```diff
@@ -90,11 +115,11 @@ export default {
     }
   },
   components: {
-    Dropdown,
-    DropdownHeader,
-    DropdownItem,
+    Dropdown : patchedDropdown,
```
Comment (ChristianMurphy) [inline notification-portlet-webcomponents/notification-icon/src/components/NotificationIcon.vue]:
```suggestion
    Dropdown: patchedDropdown,
```

File: notification-portlet-webcomponents/notification-icon/src/components/NotificationIcon.vue [hunk 3]
```diff
@@ -90,11 +115,11 @@ export default {
...
+    DropdownHeader : patchedDropdownHeader,
```
Comment (ChristianMurphy) [inline notification-portlet-webcomponents/notification-icon/src/components/NotificationIcon.vue]:
```suggestion
    DropdownHeader: patchedDropdownHeader,
```

File: notification-portlet-webcomponents/notification-icon/src/components/NotificationIcon.vue [hunk 4]
```diff
@@ -90,11 +115,11 @@ export default {
...
+    DropdownItem : patchedDropdownItem,
```
Comment (ChristianMurphy) [inline notification-portlet-webcomponents/notification-icon/src/components/NotificationIcon.vue]:
```suggestion
    DropdownItem: patchedDropdownItem,
```

File: notification-portlet-webcomponents/notification-icon/src/components/ieDropdown.vue [hunk 5]
```diff
@@ -0,0 +1,23 @@
+<template>
+  <div class="dropdown">
+    <button class="btn btn-secondary dropdown-toggle"
+            type="button"
+            id="dropdownMenuButton"
+            data-toggle="dropdown"
+            aria-haspopup="true"
+            aria-expanded="false"
```
Comment (cousquer) [inline notification-portlet-webcomponents/notification-icon/src/components/ieDropdown.vue]: heya @mindblender 👋 , just a small comment about aria. you need to provide a way to set this attribute `aria-expanded` to `true`, when the button is activated. And toggle its state to `false`, when dropdown is hidden.

**Pull request merged**